### PR TITLE
Add requesting and saving of raw data from Clarius

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
Raw data can now be saved as a .tar file using the SAVE_RAW_DATA command.

Example usage:
```
  <Command Name="SaveRawData" OutputFilename="Test.tar" RawDataLastNSeconds="-1"></Command> ==> Saves all available raw data to Test.tar
  <Command Name="SaveRawData" RawDataLastNSeconds="10"></Command> ==> Saves the last 10 seconds of raw data to MMDDYY_HHMMSS_Raw.tar
```

### CODE REVIEW

**Comment by adamrankin** [comment]
Add .gz capabilities?

File: src/PlusDataCollection/Clarius/vtkPlusClarius.cxx [hunk 1]
```diff
@@ -561,3 +566,127 @@ PlusStatus vtkPlusClarius::WritePosesToCsv(const ClariusImageInfo *nfo, int npos
...
+  LOG_INFO("Receiving " << dataSize << " bytes of raw data");
+
+  ClariusReturnFn returnFunction = (ClariusReturnFn)(&vtkPlusClarius::RawDataReadFn);
+  this->AllocateRawData(dataSize);
+  clariusReadRawData(&this->RawDataPointer, returnFunction);
+  return PLUS_SUCCESS;
+}
+
+//----------------------------------------------------------------------------
+void vtkPlusClarius::RawDataReadFn(int retCode)
```
Comment (adamrankin) [inline src/PlusDataCollection/Clarius/vtkPlusClarius.cxx]: RawDataWriteFn?

Comment (Sunderlandkyl) [reply src/PlusDataCollection/Clarius/vtkPlusClarius.cxx]: Good point, changed!

**Comment by Sunderlandkyl** [comment]
Added ability to save raw data with gzip compression.

**Comment by adamrankin** [comment]
Looks good to me

**Comment by Sunderlandkyl** [comment]
Great, thanks!

**Pull request merged**